articles/aks/howto-deploy-java-liberty-app.md

```diff
@@ -11,7 +11,7 @@ keywords: java, jakartaee, javaee, microprofile, open-liberty, websphere-liberty
 
 # Deploy a Java application with Open Liberty/WebSphere Liberty on an Azure Kubernetes Service cluster
 
-This guide demonstrates how to run your Java, Java EE, [Jakarta EE](https://jakarta.ee/), or [MicroProfile](https://microprofile.io/) application on the Open Liberty/WebSphere Liberty runtime and then deploy the containerized application to an Azure Kubernetes Service (AKS) cluster using the Open Liberty Operator. This article will walk you through preparing a Liberty application, building the application Docker image and running the containerized application on an AKS cluster.  For more details on Open Liberty, see [the Open Liberty project page](https://openliberty.io/). For more details on IBM WebSphere Liberty see [the WebSphere Liberty product page](https://www.ibm.com/cloud/websphere-liberty).
+This guide demonstrates how to run your Java, Java EE, [Jakarta EE](https://jakarta.ee/), or [MicroProfile](https://microprofile.io/) application on the Open Liberty/WebSphere Liberty runtime and then deploy the containerized application to an Azure Kubernetes Service (AKS) cluster using the Open Liberty Operator. The Open Liberty Operator simplifies the deployment and management of applications running on Open Liberty Kubernetes clusters. You can also perform Day-2 operations such as gathering traces and dumps using the operator. This article will walk you through preparing a Liberty application, building the application Docker image and running the containerized application on an AKS cluster.  For more details on Open Liberty, see [the Open Liberty project page](https://openliberty.io/). For more details on IBM WebSphere Liberty see [the WebSphere Liberty product page](https://www.ibm.com/cloud/websphere-liberty).
 
 [!INCLUDE [quickstarts-free-trial-note](../../includes/quickstarts-free-trial-note.md)]
```

Comment:
> •	If **you are** running the commands in this guide locally (instead of Azure Cloud Shell)
As you can see, the statement with the recommended wording change is from an **included** article, which is also largely included in other Azure docs. So I suggest keep it unchanged as for now.

Reply:
OK, happy to keep unchanged